src/test/java/components/Table.java

```diff
@@ -19,13 +19,4 @@ public ElementsCollection getTableRows() {
     public SelenideElement getRow(int indexOfRow) {
         return getTableRows().get(indexOfRow);
     }
-
-    public ElementsCollection getCellsForRow(int rowPosition) {
-        return rows.shouldBe(sizeGreaterThanOrEqual(rowPosition + 1))
-                .get(rowPosition).$$("td");
-    }
-
-    public SelenideElement getFirstRow() {
-        return getTableRows().first();
-    }
```

Comment:
I would like to leave this code as it is for future improvements

Reply:
You can leave it. And in any case you will have it in your **git** history. 
But, in general, all comment blocks and functions that is never used in your actual program - such blocks should be deleted from repository )